docs/workflow/testing/libraries/testing.md

```diff
@@ -2,22 +2,37 @@
 
 We use the OSS testing framework [xunit](http://xunit.github.io/).
 
-To build the tests and run them you can call the libraries build script.
+To build the tests and run them you can call the libraries build script. **For every architecture it's mandatory to build a runtime (clr or mono)** even if you want to run libs tests only.
 
 **Examples**
 - The following shows how to build only the tests but not run them:
 ```
 build.cmd/sh -subset libs.tests
 ```
 
+- The following builds and runs all tests using clr:
+```
+build.cmd/sh -subset clr+libs.tests -test
+```
+
+- The following builds and runs all tests using mono:
+```
+build.cmd/sh -subset mono+libs.tests -test
+```
+
```

Comment:
Ideally this would be true but I don't think it is today. The built runtime assets aren't automatically binplaced into the libraries testhost folder and instead are pulled in when the libs.pretest subset is invoked.

Reply:
Did you mean to specify: `clr+libs+libs.tests`?

Reply:
@ViktorHofer so the command should be:

```diff
+ build.cmd/sh -subset clr+libs.pretest
- build.cmd/sh -subset clr+libs.tests -test
```

?

Reply:
This would only be the command for incremental runtime builds, which I don't think we have documented anywhere (as it's complicated). I would just document the "fresh build" scenario which is: `build.cmd/sh clr+libs+libs.tests -test -rc Release`.

Reply:
Resolved by @safern's suggestion.